test/e2e/edu-helpers.js

```diff
@@ -18,26 +18,47 @@ function initApplicationSubmitMock(form) {
   });
 }
 
-function completeVeteranInformation(client, data, onlyRequiredFields) {
+function completeVeteranInformation(client, data, onlyRequiredFields, root = 'root') {
+  // Use relativeFullname if necessary
```

Comment:
You make an excellent point.

The json has `view:currentSponsorInformation` -> `relativeFullName` but the `formConfig` has `view:currentSponsorInformation` -> `veteranFullName` which makes more sense. I'll update the json to match the config.